14/events/index.html

```diff
@@ -0,0 +1,1234 @@
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href="#" onclick=
+                                                "desc('kalakshetra','rangoli')"
+                                                style="width:100px">Rangoli</a>
+                                                </li>
+
+
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href="#" onclick=
+                                                "desc('kalakshetra','graffiti')"
+                                                style=
+                                                "width:100px">Graffiti</a>
+                                                </li>
+
+
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href="#" onclick=
+                                                "desc('kalakshetra','paperdressing')"
+                                                style="width:100px">Paper
+                                                Dressing</a>
+                                                </li>
+
+
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href="#" onclick=
+                                                "desc('kalakshetra','bagdesign')"
+                                                style="width:100px">T-shirt
+                                                Design</a>
+                                                </li>
+                                                <!--    <li><a href="#" onclick="desc('kalakshetra','mapdesign')">Mag Design</a></li>
+-->
+
+
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href="#" onclick=
+                                                "desc('kalakshetra','potpainting')"
+                                                style="width:100px">Pot
+    
```

Comment:
Oh lord....

@polybuildr , @hharchani , @nisargjhaveri 

Should we agree to edit and clean out the <br/> and &nsbsp; atleast?
Rewrite of this  section should not be too long either.